### ISSUE TITLE
[iOS] Example project: Build Failed

### ISSUE THREAD

**Issue by irbisdev** [infobip/mobile-messaging-react-native-plugin#29]
Tried to build example project from repo on iOS. 

**Before build:**
```
> yarn
> cd ios
> pod install
```

**Result:**

<img width="369" alt="Screenshot 2021-07-28 at 22 56 29" src="https://user-images.githubusercontent.com/4831278/127387793-9d221494-27ce-400f-a39c-af95d2932e90.png">

**Environment:**
Xcode 12.5.1
CocoaPods 1.10.1

---

**One of solution:**

`ios/Podfile`
```Ruby
......

use_frameworks!

target 'Example' do

  ......

  $static_framework = [
    'react-native-safe-area-context',
    'React-RCTText',
    'RNReanimated'
  ]
    
  pre_install do |installer|
    Pod::Installer::Xcode::TargetValidator.send(:define_method, :verify_no_static_framework_transitive_dependencies) {}

    installer.pod_targets.each do |pod|
      if $static_framework.include?(pod.name)
        def pod.build_type;
          Pod::BuildType.static_library
        end
      end
    end
  end
end
```

**Comment by riskpp** [comment]
@irbisdev Thank you! 
We will check and update Example.

BR, Olga

**Comment by riskpp** [comment]
Didn't reproduce this issue, however found different one, some time ago by mistake `#import <React/RCTLinkingManager.h>` was removed from [AppDelegate.m](https://github.com/infobip/mobile-messaging-react-native-plugin/blob/master/Example/ios/Example/AppDelegate.m#L5)

I've updated it.

About issue you mentioned, would be nice to try to make following steps:
1.  `pod deintegrate`, `pod install` with closed Xcode
2.  delete derived data and do `Product->Clean Build Folder` in Xcode
3.  try to build again

Also we have guide [How to start Example app](https://github.com/infobip/mobile-messaging-react-native-plugin/wiki/How-to-start-Example-app) and we don't use `yarn`, but `npm install`, but it shouldn't be the cause of this issue, I've checked.

BR, Olga
